test/integration/extension_discovery_integration_test.cc

```diff
@@ -322,6 +382,76 @@ TEST_P(ExtensionDiscoveryIntegrationTest, BasicDefaultMatcher) {
   EXPECT_EQ("200", response->headers().getStatusValue());
 }
 
+TEST_P(ExtensionDiscoveryIntegrationTest, ReuseExtensionConfig) {
```

Comment:
Can you add short comments above each test case explaining what they test (pointing to the regression)? Thanks.

Reply:
Done. Thank you for the review.